cypress/integration/tests/developer/reports/configure.report.test.ts

```diff
@@ -0,0 +1,19 @@
+import { login, preservecookies } from "../../../../utils/utils";
+import { ReportConfig } from "../../../models/developer/reports/reportConfig";
+
+describe("This test set is configuring reports", function () {
+    before("Login and create test data", function () {
+        login();
+    });
+
+    beforeEach("Persist session", function () {
+        // Save the session and token cookie for maintaining one login session
+        preservecookies();
+    });
+
+    it("This test is enabling download of HTML and CSV reports", function () {
+        let reportConfig = ReportConfig.getInstance();
```

Comment:
**nitpick:** This variable could be a constant as its never reassigned after declared

Reply:
This is not correct, I will change fields of the object when changing configuration. That is why it is `let` but not `const`

Reply:
I understand, anyway, as long as you only change properties instead of replacing the full `reportConfig` variable it can be a `const`. But maybe that's a matter of personal taste, the PR is good as it is